opslevel/import_utils.go

```diff
@@ -0,0 +1,59 @@
+package opslevel
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/opslevel/opslevel-go/v2024"
+)
+
+func extractContactFromContacts(contactId opslevel.ID, contacts []opslevel.Contact) *opslevel.Contact {
```

Comment:
These `extract` functions have almost the same exact code, I think a new resource interface in opslevel-go + generics would solve. I think there is a lot of code in the repo so far that repeats logic in a similar manner that we should start avoiding.

Reply:
A generic ContainsID() func would be nice. Would be a good future ticket to tackle if you feel strongly about this